cumulus_library/.sqlfluff

```diff
@@ -16,21 +16,25 @@ ignore_comment_lines = true
 capitalisation_policy = upper
 
 [sqlfluff:templater:jinja:context]
+code_systems = ["http://snomed.info/sct", "http://hl7.org/fhir/sid/icd-10-cm"]
+col_type_list = ["a string","b string"]
+cc_columns = [{"name": "baz", "is_array": True}, {"name": "foobar", "is_array": False}]
+cc_column = 'code'
+dataset = [["foo","foo"],["bar","bar"]]
+ext_systems = ["omb", "text"]
+field = 'column_name'
+fhir_extension = fhir_extension
+id = 'id'
+prefix = Test
 schema_name = test_schema
+source_table = source_table
+source_id = source_id
+table_cols = ["a","b"]
 table_name = test_table
+target_col_prefix = prefix
+target_table = target_table
+unnests = [{"source col": "g", "table_alias": "i", "row_alias":"j"}, {"source col": "k", "table_alias": "l", "row_alias":"m"},]
+view_cols = ["c","d"]
```

Comment:
cc_columns and unnests are the only new sqlfluff aliases here - otherwise, this is just trying to apply order to chaos.